disnake/ext/commands/base_core.py

```diff
@@ -380,8 +380,8 @@ def get_cooldown_retry_after(self, inter: ApplicationCommandInteraction) -> floa
 
         return 0.0
 
+    # This method isn't really usable in this class, but it's usable in subclasses.
     async def invoke(self, inter: ApplicationCommandInteraction, *args, **kwargs) -> None:
-        """This method isn't really usable in this class, but it's usable in subclasses."""
```

Comment:
this should continue to be documented

Reply:
`Command.invoke` and `Group.invoke` are also not documented, all of these should be implementation details.

Reply:
sounds fair.

Reply:
we should mention something about this method now being undocumented though.

Reply:
https://github.com/DisnakeDev/disnake/pull/961/commits/87d27a96e5ead52c350b8944b76d88a1f996808c